.script/tests/KqlvalidationsTests/YamlFilesTestData/YamlFilesLoader.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Kqlvalidations.Tests
+{
+    public abstract class YamlFilesLoader
+    {
+        protected List<string> DirectoryPaths { get; }
```

Comment:
it can be just private readonly memeber - why do we need Property and why protected ?